recipe-client-addon/lib/Errors.jsm

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Module of shared error subclasses used within the recipe client add-on.
+ */
+
+"use strict";
+
+this.EXPORTED_SYMBOLS = ["InvalidSignatureError"];
+
+class InvalidSignatureError extends Error {}
```

Comment:
JSMs are unfortunately not as cheap as they should be. Does this really need its own file? Maybe it could live as a property on either NormandyApi or RecipeRunner and be reused from there?